test.py

```diff
@@ -370,7 +370,7 @@ def main():
     # We are using a token that can be validated by DataEdge
     edge_payload = {'destinationFormat': 'plain'}
     edge_headers = {'Authorization': f'Bearer {token}'}  # No token no permissions
-    dataedge_url = f"http://{config['dataedge_address']}:{config['dataedge_port']}/files/{fileID}"
+    dataedge_url = f"http://{config['dataedge_address']}:{config['dataedge_port']}/files/EGAF{stableID}"
```

Comment:
the `stableID` needs to be `EGAF<about 10digits>` also in the database, so when you send the message with the `stableID` send it there in that form, and here just reuse it.